packages/js/src/core/fetch-token.ts

```diff
@@ -45,20 +42,13 @@ export const fetchTokenByRefreshToken = async (
   }
 
   const requester = createRequester(fetchFunction);
-  const response = await requester(tokenEndPoint, {
+  const tokenSnakeCaseResponse = await requester<TokenSnakeCaseResponse>(tokenEndPoint, {
     method: 'POST',
     headers: ContentType.formUrlEncoded,
     body: parameters,
   });
 
-  assert(response, TokenResponseSchema);
-  const { access_token, refresh_token, id_token, scope: response_scope, expires_in } = response;
```

Comment:
I think we do have to keep the assert? Otherwise if the actual response fails to meet our typescript types in runtime, we don't have a guard to prevent it from causing further issues

Reply:
OK, based on the latest team discussion, we can assume the server response is always in correct type, as it is totally controllable. Thus the assert is not necessary any more.

Reply:
Unnecessary. Logto Core server is under our control, and we assume its response will have  the correct structure or the code should fail directly.